cgltf_write.h

```diff
@@ -392,7 +392,7 @@ static void cgltf_write_mesh(cgltf_write_context* context, const cgltf_mesh* mes
 
 	if (mesh->weights_count > 0)
 	{
-		cgltf_write_floatarrayprop(context, "weights", mesh->weights, mesh->weights_count);
+		cgltf_write_floatarrayprop(context, "weights", mesh->weights, (int)mesh->weights_count);
```

Comment:
Instead of casting to int here, maybe we should change the decl of `cgltf_write_floatarrayprop` to take a cgltf_size.

Reply:
Agreed. @LxLasso can you change that?

Reply:
Also, I think using `cgltf_size` is more in line with the rest of the code than `size_t`. So that seems fine to me.

Reply:
Agreed, I'll change last parameter of  `cgltf_write_floatarrayprop` to cgltf_size.

Reply:
The reason I changed the variable from cgltf_size to size_t was that strlen returns size_t and the variable simply catches the result of strlen. The implicit cast was causing a warning (which seems a bit odd now that I think about it). I'll change it back to cgltf_size and add an explicit cast instead!